tests/mock_robotd.py

```diff
@@ -138,12 +143,21 @@ class MockCamera(Camera):
     board_type_id = 'camera'
 
     def __init__(self, name, node, camera):
-        node['DEVNAME'] = "/" + name
+        node['DEVNAME'] = name
         self.serial = name
         # Create a camera with a file camera instead of a real camera
         super().__init__(node, camera)
 
 
+class MockGameState(GameState):
+    board_type_id = 'game'
+
+    def __init__(self, name, node):
+        super().__init__()
```

Comment:
the constructor for `GameState` takes parameters, you should be passing these in, or at least passing a stub